### ISSUE TITLE
Transitions using with_cfg seems to drop exec_groups from test targets

### ISSUE THREAD

**Issue by bedstefaer** [fmeum/with_cfg.bzl#133]
I am using with_cfg to transition a target to run on arm64. I have made a macro to wrap this behavior.
I am using c++ and the built-in execution group "test" to decide how to run.

example:
~~~python
_cc_arm_test_builder = with_cfg(native.cc_test)
_cc_arm_test_builder.set(
    "platforms",
    [Label("@//bazel/toolchains/config:platform-clang-aarch64")],
)

cc_arm_test, cc_arm_test_internal = _cc_arm_test_builder.build()

def cc_test(name, run_on_arm=False, **kwargs):
    # here is maps through correctly.
    native.cc_test(
        name = name,
        exec_properties = {
            "test.arch": "arm64",
        },
        **kwargs,
    )

    if run_on_arm:
        # error: Tried to set properties for non-existent exec groups: test.
        cc_arm_test(
            name = name,
            exec_properties = {
                "test.arch": "arm64",
            },
            **kwargs,
        )
~~~

Is there a workaround for mapping execution groups through? I am using with_cfg version 0.4.2

**Comment by fmeum** [comment]
Could you test whether https://github.com/fmeum/with_cfg.bzl/pull/134 fixes this (for example using a `git_override` with Bzlmod)?

**Comment by bedstefaer** [comment]
Just tested the patch. Works fine, thank you!

I saw you included it in 0.6.1 so ill just bump to this :)
